opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -45,4 +161,11 @@
      * @see Span#context()
      */
     void finish(long finishMicros);
+
+    /**
+     * Defined as equivalent to calling {@code tracer.scopeManager().activate(this)} on the {@link Tracer} instance
+     * that started this {@link Span} instance.
+     */
+    Scope activate();
```

Comment:
this could be removed for the next merge... arguably should be there since the Tracer won't nec be available, but in practice it (the Tracer) always seems to be.

Reply:
Agreed, this could as well go away. Nice to have, but not something really important/required.